README.md

```diff
@@ -57,11 +60,16 @@ In order to test the provider, you can simply run `make test`.
 
```

Comment:
Since we're making changes here, let's maybe remove "simply" from this?

Reply:
Done!